demo_nodes_cpp/src/services/add_two_ints_client.cpp

```diff
@@ -45,7 +55,17 @@ int main(int argc, char ** argv)
 
   auto node = rclcpp::Node::make_shared("add_two_ints_client");
 
-  auto client = node->create_client<example_interfaces::srv::AddTwoInts>("add_two_ints");
+  if (cli_option_exist(argv, argv+argc, "-h")) {
+    print_usage();
+    return 0;
+  }
+
+  auto topic = cli_get_option(argv, argv+argc, "-t");
+  if (!topic) {
+    topic = const_cast<char *>("add_two_ints");
```

Comment:
This seems dubious. What if the input to `cli_get_option` is really a string literal...